msg/tools/uorb_rtps_classifier.py

```diff
@@ -117,17 +117,20 @@ def parse_yaml_msg_id_file(yaml_file):
     parser.add_argument("-m", "--topic-msg-dir", dest='msgdir', type=str,
                         help="Topics message dir, by default msg/", default="msg")
     parser.add_argument("-y", "--rtps-ids-file", dest='yaml_file', type=str,
-                        help="RTPS msg IDs definition file, relative to the msg_dir, by default tools/uorb_rtps_message_ids.yaml",
-                        default="tools/uorb_rtps_message_ids.yaml")
+                        help="RTPS msg IDs definition file absolute path, by default use relative path to msg, tools/uorb_rtps_message_ids.yaml",
+                        default='tools/uorb_rtps_message_ids.yaml')
 
     # Parse arguments
     args = parser.parse_args()
 
     msg_folder = args.msgdir
     if args.msgdir == 'msg':
         msg_folder = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
-    classifier = Classifier(os.path.join(
-        msg_folder, args.yaml_file), msg_folder)
+    if args.yaml_file != 'tools/uorb_rtps_message_ids.yaml':
```

Comment:
Can you also use the absolute path check here?

Reply:
Sure. Done!